test/units/parsing/vault/test_vault_editor.py

```diff
@@ -207,7 +208,7 @@ def test_rekey_migration(self):
 
         os.unlink(v10_file.name)
 
-        assert vl.cipher_name == "AES256", "wrong cipher name set after rekey: %s" % vl.cipher_name
+        assert re.search(';AES256',fdata), "wrong cipher name set after rekey"
```

Comment:
You could do the same `';AES256' in fdata` check here as in test_vault_editor.py, and avoid the `re` module.